aries_cloudagent/verifier/base.py

```diff
@@ -0,0 +1,39 @@
+"""Base Verifier class."""
```

Comment:
I don't think this is meant to be added?

Reply:
Right: it's an artifact